CHANGELOG.md

```diff
@@ -1,5 +1,68 @@
 # Version changelog
 
+## Release v0.28.0
+
+### Improvements and new features
+
+ * Better error message when private link enabled workspaces reject requests ([#647](https://github.com/databricks/databricks-sdk-py/pull/647)).
+
+### API Changes:
```

Comment:
We could move similar sections together for better readability like all Added together, Removed, Renamed, Changed.